### DISCUSSION TITLE
pm.environment.name to Bruno

### DISCUSSION THREAD

**Discussion by JosephGuzmanL** [usebruno/bruno#2516] [Q&A]
There is a way to obtain the environment name for a request when a environment is selected? I mean, if I switch to LOCAL, to QA environments, i got this names in the script? with postman the command is `pm.environment.name`, what is the command for Bruno? I did not found anything in the documentation

**Comment by Its-treason** [comment]
Yes, `bru.getEnvName()` will return the current environment name or null if no environment is selected.

**Comment by baptpln** [comment]
We should add this command in the docs and in the translator
